index.php

```diff
@@ -0,0 +1,43 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1,user-scalable=no">
+    <title>Ballers Visitors</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <link rel="stylesheet" href="assets/css/styles.css">
+  </head>
+  <body>
+    <div class="container">
+        <div class="form-container col-md-6 col-sm-12">
+          <div class="text-center">
+            <img src="assets/img/logo.png" alt="" class="logo">
+          </div>
+
+          <h3 class="land text-center">Become a Landlord</h3>
+          <p class="text-center">Join a community of insightful minds where you can own a home with ease.</p>
+          <br>
+          <p class="text-center">Pls fill out your details below to be notified when we launch.</p>
+
+          <form>
+            <div class="form-group">
+              <small>Fullname</small>
```

Comment:
Nit: This should be in a label
```suggestion
              <label for="fullName"><small>Fullname</small></label>
```

Reply:
Kindly fix the other lines below too.